### PR TITLE
Saas 11.5 description yti

### PR DESCRIPTION
**Pull request by tivisse**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by mart-e** [comment]
:tada: lets merge that before new modules lands in the stable version !

File: addons/base_import/models/test_models.py [hunk 1]
```diff
@@ -89,13 +89,15 @@ class PreviewModel(models.Model):
 
 class FloatModel(models.Model):
     _name = name('float')
+    _description = 'Tests: Base Import Model Float'
```
Comment (xmo-odoo) [inline addons/base_import/models/test_models.py]: C'est de la high-value description, ya pas moyen de dire que ce modèle a pas besoin d'une description à la place?

Comment (tivisse) [reply addons/base_import/models/test_models.py]: Ouais, ou alors tu mets le nom technique à la place.

**Comment by tivisse** [comment]
@mart-e "new modules lands in the stable version !" Have you ever read the stable policy???

Please check https://github.com/odoo/odoo/wiki/Contributing#what-does-stable-mean

**Comment by mart-e** [comment]
@tivisse fun fact, did you know the time it took me to write this message, 2 new modules have been merged in saas-11.5?

**Pull request merged**